.happy/terraform/modules/sfn/main.tf

```diff
@@ -81,14 +86,69 @@ resource "aws_sfn_state_machine" "state_machine" {
             "Next": "HandleErrors",
             "ResultPath": "$.error"
           }
-        ]
+        ],
+        "ResultPath": "$.batch"
```

Comment:
I'm a little confused as to where `.batch` is coming from. What is producing/populating this output?

Reply:
I understand the nature of `ResultPath` well enough I think. I am just confused as to when/where `.batch` is being assigned on the path.

Reply:
Oh, this is actually _assigning_ the output as being on `.batch`. Nvm.

Reply:
If the original input was
```
{
 "foo": "bar",
}
```

The result would go
```
{
 "foo": "bar",
 "batch": "result
}
```

Reply:
the response that populates it is from here https://github.com/chanzuckerberg/single-cell-data-portal/blob/1ea8e43f4e0c8721e545dcf8532b969e7bd9211c/backend/layers/processing/process_download.py#L193

Reply:
Yep